src/context.ts

```diff
@@ -856,7 +856,7 @@ export class Context<U extends Deunionize<tg.Update> = tg.Update> {
    */
   sendChatAction(
     action: Shorthand<'sendChatAction'>[0],
-    extra?: tt.ExtraSendChatAction
+    extra?: Omit<tt.ExtraSendChatAction, 'intervalDuration'>
```

Comment:
Revert this to the original form since you seem to have reverted changes to `ExtraSendChatAction`.